aya/src/obj/btf/btf.rs

```diff
@@ -143,7 +143,7 @@ impl Btf {
         }
 
         let mut data = &data[base..base + type_len];
-        let mut types = vec![BtfType::Unknown];
+        let mut types = vec![];
         while !data.is_empty() {
```

Comment:
The first type must always be `Unknown` or it will break `type_by_id`

Reply:
fixed!